certified-connectors/tegolySIGN/apiDefinition.swagger.json

```diff
@@ -0,0 +1,164 @@
+      "x-ms-notification-content": {
+        "schema": {
+          "type": "object",
+          "properties": {
+            "name": {
+              "type": "string",
+              "description": "Name of the PDF document"
+            },
+            "document": {
+              "type": "string",
+              "format": "byte",
+              "description": "The signed PDF document"
+            },
+            "certificate": {
+              "type": "string",
+              "format": "byte",
+              "description": "The certificate for the signed doucment"
+            }
+          }
+        },
+        "description": "Returns the signed document, name and certificate"
+      },
+      "post": {
+        "responses": {
+          "default": {
+            "description": "default",
+            "schema": {
+              "type": "object",
+              "properties": {}
+            }
+          }
+        },
+        "summary": "When a document is completely signed",
+        "operationId": "CompletelySigned",
+        "x-ms-trigger": "single",
+        "parameters": [
+          {
+            "name": "body",
+            "in": "body",
+            "required": false,
+            "schema": {
+              "type": "object",
+              "properties": {
+                "name": {
+                  "type": "string",
+                  "description": "name",
+                  "title": "",
+                  "x-ms-visibility": "advanced"
+                },
+                "delivery_url": {
+                  "type": "string",
+                  "description": "delivery_url",
+                  "title": "",
+                  "x-ms-visibility": "internal",
+                  "default": "document.callback",
+                  "x-ms-notification-url": true
+                }
+              },
+              "required": [
+                "delivery_url",
+                "name"
+              ],
+              "x-ms-visibility": "advanced"
+            },
+            "x-ms-visibility": "advanced"
+          }
+        ],
+        "description": "When a document is fully signed and imported via the tegolySIGN connector, the document and certificate are transferred."
+      }
+    },
+    "/api/webhook/action": {},
+    "/api/webhook/create-draft": {
+      "post": {
+        "security": [
+          {
+            "oauth2_auth": []
+          }
+        ],
+        "responses": {
+          "default": {
+            "description": "default",
+            "schema": {
+              "type": "object",
+              "properties": {}
+            }
+          }
+        },
+        "summary": "Import a file to tegolySIGN",
+        "operationId": "ImportPDF",
+        "consumes": [
+          "multipart/form-data"
+        ],
+        "parameters": [
+          {
+            "name": "file",
+            "in": "formData",
+            "description": "The uploaded file data",
+            "required": true,
+            "type": "file"
```

Comment:
@JanGosda The 'x-ms-summary' property is required, please add to the parameters.